src/main/scala/com/gu/support/workers/lambdas/FailureHandler.scala

```diff
@@ -39,34 +39,24 @@ class FailureHandler(emailService: EmailService)
   private def handleError(state: FailureHandlerState, error: Option[ExecutionError], requestInfo: RequestInfo) = {
     SafeLogger.info(s"Attempting to handle error $error")
     error.flatMap(extractUnderlyingError) match {
-      case Some(ZuoraErrorResponse(_, List(ze @ ZuoraError("TRANSACTION_FAILED", _)))) => returnState(
+      case Some(ZuoraErrorResponse(_, List(ze @ ZuoraError("TRANSACTION_FAILED", _)))) => exitHandler(
         state,
         requestInfo.appendMessage(s"Zuora reported a payment failure: $ze")
       )
-      case Some(se @ StripeError("card_error", _, _, _, _)) => returnState(
+      case Some(se @ StripeError("card_error", _, _, _, _)) => exitHandler(
         state,
         requestInfo.appendMessage(s"Stripe reported a payment failure: ${se.getMessage}")
       )
-      case _ => returnState(state, requestInfo.copy(failed = true),
-        "There was an error processing your payment. Please\u00a0try\u00a0again\u00a0later.")
+      case _ => exitHandler(state, requestInfo.copy(failed = true))
     }
   }
 
-  private def returnState(
+  private def exitHandler(
     state: FailureHandlerState,
-    requestInfo: RequestInfo,
-    message: String = "There was an error processing your payment. Please\u00a0try\u00a0again."
```

Comment:
We always just display the default failure message client-side, so we're not really losing anything by removing this state at the moment...

I'd like to re-introduce a CheckoutFailure state in a separate PR, which can be used to pass more specific errors back to the client.